DEVELOPMENT.md

```diff
@@ -118,7 +118,7 @@ By default, the Remix `rollup` build will strip any `console.debug` calls to avo
 REMIX_DEBUG=true yarn watch
 ```
 
-**`REMIX_LOCAL_DEV_OUTPUT_DIRECTORY`**
+**`LOCAL_BUILD_DIRECTORY`**
```

Comment:
I do think its nice to have a framework-specific prefix for env vars to avoid conflicts with existing env vars. Since the company name is Remix, I think `REMIX_` is good. Maybe there's something better.